### PR TITLE
Added your meeting buddy

### LINKED ISSUE #702: Your meeting buddy
# Aim

This script aims to join the google meetings at a **_scheduled_** time in our absence. 

# Details

- Auto joining meets in our absence.
- joining at a scheduled time.
- Complete discussion is recorded in local machine.

### PR DESCRIPTION
**Pull request by Yuvraj-kadale**
# Description
Added "Your meeting buddy", This buddy will help you join the meetings in your absence at a scheduled time saved in your google calendar. 

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Your Meeting Buddy/Gmeet_buddy.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import keyboard
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.chrome.options import Options
+
+#joining Credentials
+
+username="email@gmail.com"          #enter your mail id
+password="This is my password"      #enter your password
+
```
Comment (santushtisharma10) [inline Your Meeting Buddy/Gmeet_buddy.py:14]: @Yuvraj-kadale instead of harcoding the credential, take the input from the user

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]

**Comment by santushtisharma10** [comment]
@kaustubhgupta i am facing error while running the script on my local machine. Please confirm if the script is working or not.

**Comment by Yuvraj-kadale** [comment]
> @kaustubhgupta i am facing error while running the script on my local machine. Please confirm if the script is working or not.

Could you please let me know the error You are getting?

**Comment by Yuvraj-kadale** [comment]
@santushtisharma10 could you please sure that you have downloaded chrome driver, and have defined the correct path in the code. I have tried running the script in my local machine. The working can be viewed [here](https://youtu.be/auJaJZ_h0gE)

**Comment by santushtisharma10** [comment]
> @santushtisharma10 could you please sure that you have downloaded chrome driver, and have defined the correct path in the code. I have tried running the script in my local machine. The working can be viewed [here](https://youtu.be/auJaJZ_h0gE)

Yes i have downloaded it and the path was also correct. And i saw the working of your script ,but still getting an error on my local system, so will confirm it with @kaustubhgupta . If he is able to run it successfully on his local system, then I will approve the PR.

File: Your Meeting Buddy/Gmeet_buddy.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+opt.add_argument("--disable-extensions")
+
+# Passing the argument "1" to allow and "2" to block
+
+opt.add_experimental_option("prefs", { \
+    "profile.default_content_setting_values.media_stream_mic": 1, 
+    "profile.default_content_setting_values.media_stream_camera": 1, 
+    "profile.default_content_setting_values.notifications": 1 
+      })
+browser =webdriver.Chrome(chrome_options=opt,executable_path="C:\\Users\\path where your chrome driver is saved")   #Enter the path of your chrome driver
```
Comment (kaustubhgupta) [inline Your Meeting Buddy/Gmeet_buddy.py:29]: take input for the web driver path from user

File: Your Meeting Buddy/Readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
+# Your Meeting Buddy
+This buddy will help you to attain any online meetings scheduled on google meet in your absence and will record the meet for you. So as to review/revise whenever required.
+
+## Required Setup
+Inorder to automate the meeting process you are required to install the following drivers
+### Quick links to the driver:
```
Comment (kaustubhgupta) [inline Your Meeting Buddy/Readme.md:7]:
```suggestion
## Quick links to the driver:
```

File: Your Meeting Buddy/Readme.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+<ul>Chrome Driver:<a href="https://chromedriver.chromium.org/downloads" target="_blank" >Download</a></ul>
+<ul>Firefox Driver:<a href="https://github.com/mozilla/geckodriver/releases" target="_blank">Download</a></ul>
+
+### How to wake my buddy?
```
Comment (kaustubhgupta) [inline Your Meeting Buddy/Readme.md:11]:
```suggestion
## How to wake my buddy?
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
@santushtisharma10 The script works, @Yuvraj-kadale Here are some suggested changes:

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@Kushal997-das

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..Great Job @Yuvraj-kadale

**Pull request merged**